platform/W4.roc

```diff
@@ -68,6 +109,17 @@ Mouse : {
     middle : Bool,
 }
 
+## Represents the current state of [Netplay](https://wasm4.org/docs/guides/multiplayer#netplay). 
+##
+## > Netplay connects gamepad inputs over the Internet using WebRTC
+## 
```

Comment:
Not sure if these extra comments should go here on on the `readNetplay` function, but I think we should include this extra info somewhere:
```suggestion
## > Netplay connects gamepad inputs over the Internet using WebRTC
## 
## Note: All WASM-4 games that support local multiplayer automatically support netplay.
## There are no additional steps developers need to take to make their games netplay-ready.
## It can be used to implement advanced features such as non-shared screen multiplayer
```